oaknational/Oak-Web-Application
# feat: card image button icon variants

## Description

- Adds Components for card variant with image / icon and button
- Adds CardImage and CardTitle component
- Adds margin and color convenience props to typography

_Next Image was causing some issues with Storybook so have reverted back to img_


![Screenshot 2022-06-08 at 12 36 23](https://user-images.githubusercontent.com/7987787/172607279-db47f50e-6b48-4e09-93c3-a77d63dfd4a3.png)



## Issue(s)

Fixes #
https://github.com/oaknational/Samara/issues/207

## How to test

Go to https://samara-storybook-ik310901d-oak-national-academy.vercel.app/
Click on Card (docs view)
You should see the component and its variations

## Screenshots



## Checklist
